src/news-feed-events/NewsFeedEvent.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from "react";
+
+/**
+ * Represents an action taken.
+ */
+export default interface NewsFeedEvent {
+  "UNIQUE ID #": string;
+  "STATE": string;
+  "COUNTY": string;
+  "ACTION DATE": string;
+  // This column doesn't exist in the fake data.
+  "SOURCE"?: string;
+  "OVERALL BUCKET": string;
+  "SUB-CATEGORY": string;
+  "ACTION TAKEN": string;
+  "# OF PEOPLE RELEASED": string;
+  "# OF PEOPLE PREVENTED FROM ENTERING": string;
+  "SUMMARY RE-WRITE": string;
+}
+
+/**
+ * Convert a TSV spreadsheet into NewsFeedEvent objects.
+ */
+export function parseNewsFeedEventsFromTsv(tsv: string) {
+  let table = tsv.split("\n").map((line) => line.split("\t"));
```

Comment:
I think something is simple is okay to get started, but I'd feel more comfortable using an actual csv/tsv library, which is going to have at least rudimentary support for escaping, character encodings, etc.

Reply:
This does appear brittle, didn't think about checking edge cases here.

I did a test of some edge cases just now and it looks like everything actually just works. Here's the sheet I made, which contains a tab and a newline within a cell, and some unicode characters:

![image](https://user-images.githubusercontent.com/1570168/77837751-9d970100-7121-11ea-8bb2-37fdec0fc9a3.png)

The TSV we get looks like:

```
a<TAB><TAB>b
<TAB>c        d<TAB>
文<TAB>e f<TAB>🤠
<TAB>fdfsd<TAB>
```

The tab has been replaced by a few spaces, and the newline has been replaced with a single space. (In the future, we should probably switch to using CSV output and use a parser if we need to actually want the newlines to stay newlines.)

As for encodings, it looks like there are no problems. The TSV is UTF-8, and `fetch`'s `text()` uses UTF-8 as well:

https://developer.mozilla.org/en-US/docs/Web/API/Body/text

Based on this, I think the current implementation is fine as is. Open to feedback, of course!

Reply:
I appreciate this test! I think a more accurate test would be one that samples actual form input. In spite of their best intentions, I imagine the people pasting form input from one sheet into another are not going to look for things like, e.g. newlines or tabs in the form input, or quotes of all kinds. I'm fine to stick with the current approach for now, but it would be good to ping Serena on this and ask her if she can bound what kinds of form input formatting we might permit, or if everything is going to be just straight up lines of text.

Reply:
> In spite of their best intentions, I imagine the people pasting form input from one sheet into another are not going to look for things like, e.g. newlines or tabs in the form input, or quotes of all kinds.

Having these in the input should not cause issues! As tested above, I think this system can handle all kinds of characters.

> I think a more accurate test would be one that samples actual form input.

+1! I'll be careful over the next few days to see if anything breaks -- if it does, we can use a parser library to address it.